packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/quanticSearchBoxInput.js

```diff
@@ -206,6 +211,12 @@ export default class QuanticSearchBoxInput extends LightningElement {
     this.input.blur();
   }
 
+  handleKeyDownOnClearButton(event) {
+    if (event.key === keys.ENTER) {
+      this.isClearButtonTargeted = true;
```

Comment:
```suggestion
      // Ignore next enter in searchbox input, to not trigger a search when we press enter on the clear button.
      this.ignoreNextEnter = true;
```
This needs to be explicit what it does, otherwise we'll never remember what or why we did this.